src/index.html

```diff
@@ -2,11 +2,416 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="logo-moyo"
+          class="logo-icon"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="product-list">
+      <div class="product-list__content">
+
+        <div class="product-list__row">
+
+          <div class="product-card" data-qa="card">
+            <div class="product-card__img-wrap">
+              <img
+                src="images/imac.jpeg"
+                alt="apple screen"
+                class="product-card__img"
+              >
+            </div>
+            <div class="product-card__description">
+              <div class="product-card__description-text">
+                Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+              </div>
+              <div class="product-card__description-code">
+                Код товара: 195434
+              </div>
```

Comment:
```suggestion
              <h4 class="product-card__description-text">
                Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
              </h4>
              <p class="product-card__description-code">
                Код товара: 195434
              </p>
```
також рекомендують в таких випадках використовувати симантичні теги